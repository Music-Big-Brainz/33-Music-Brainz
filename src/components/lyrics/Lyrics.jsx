import React from 'react';
import PropTypes from 'prop-types';

const Lyrics = ({ lyrics, song }) => (
  <div>
    <h1>{song}</h1>
    <p data-testid="lyrics">{lyrics}</p>
  </div>
);

Lyrics.propTypes = {
  song: PropTypes.string.isRequired,
  lyrics: PropTypes.string.isRequired
};

export default Lyrics;
