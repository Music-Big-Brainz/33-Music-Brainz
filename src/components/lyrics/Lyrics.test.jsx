import React from 'react';
import { render } from '@testing-library/react';
import Lyrics from './Lyrics';

describe('displays an album', () => {
  it('renders one single album', () => {
    const { asFragment } = render(
      <Lyrics 
        song="head" 
        lyrics="Song lyrics"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
