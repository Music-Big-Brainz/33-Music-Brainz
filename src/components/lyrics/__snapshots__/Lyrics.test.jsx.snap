// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays an album renders one single album 1`] = `
<DocumentFragment>
  <div>
    <h1>
      head
    </h1>
    <p
      data-testid="lyrics"
    >
      Song lyrics
    </p>
  </div>
</DocumentFragment>
`;
